packages/core-mobile/app/screens/portfolio/account/AccountItem.tsx

```diff
@@ -40,8 +41,9 @@ function AccountItem({
   const { activeNetwork } = useNetworks()
   const walletType = useSelector(selectWalletType)
   const context = useApplicationContext()
+  const tokenBlacklist = useSelector(selectTokenBlacklist)
   const accountBalance = useSelector(
-    selectBalanceTotalInCurrencyForAccount(account.index)
+    selectBalanceTotalInCurrencyForAccount(account.index, tokenBlacklist)
```

Comment:
should we move `selectTokenBlacklist` into `selectBalanceTotalInCurrencyForAccount`, since we always want to filter it out.